config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   root 'homes#index'
```

Comment:
This should be updated to whatever model you want your root to be, in your case I'm assuming Airport